vault-tool-server/src/Network/VaultTool/VaultServerProcess.hs

```diff
@@ -221,7 +220,8 @@ shutdownVaultServerProcess vs = do
 
 vaultIsRunning :: VaultAddress -> IO Bool
 vaultIsRunning addr = do
-    (vaultHealth addr >> pure True) `catches`
+    conn <- flip unauthenticatedVaultConnection addr <$> defaultManager
+    (vaultHealth conn >> pure True) `catches`
```

Comment:
This could be rewritten:
```haskell
(True <$ vaultHealth conn) `catches` ...
```

Reply:
2c42064fa561f294710ccd8b3ca290ace3d42d3f